plugins/jobs/girder_plugin_jobs/__init__.py

```diff
@@ -19,8 +19,9 @@
 
 import importlib
 
-from girder import events
+from girder import events, plugin
 from . import constants, job_rest
+from .models import job as job_model  # noqa
```

Comment:
Here we are importing the `job` model to "register" it with `ModelImporter`.  Alternatively, this could be done inside the `load` function below.